src/vuex/modules/keystore.js

```diff
@@ -16,6 +16,11 @@ export default () => {
     }
   }
 
+  const networkPrefix = localStorage
```

Comment:
too much magic. this should be a part of the network object in the API. i.e. `bech32prefix`

Reply:
Oh, I love this idea! Thank you! :+1: